Source/WebKit/GPUProcess/graphics/RemoteDisplayListRecorder.cpp

```diff
@@ -339,7 +339,7 @@ void RemoteDisplayListRecorder::drawImageBuffer(RenderingResourceIdentifier imag
         return;
     }
 
-    handleItem(DisplayList::DrawImageBuffer(imageBufferIdentifier, destinationRect, srcRect, options), *sourceImage);
+    handleItem(DisplayList::DrawImageBuffer(imageBufferIdentifier, destinationRect, srcRect, options), sourceImage.releaseNonNull());
```

Comment:
I am assuming all these are because of the change to apply() however, they don't seem like an improvement to me. It doesn't seem to avoid ref-counting churn so it is not clear why we're doing this.